en/docs/develop/product-apis/advanced-configurations.md

```diff
@@ -0,0 +1,56 @@
+# Advanced Configurations
+
+The following sections explain the advanced configurations with regard to WSO2 API Manager REST APIs.
+
+## Configuring Allowed Origins
+
+The Cross Origin Resource Sharing (CORS) `allowedOrigins` property can be used to specify the origins which should be allowed access to the Publisher, Developer Portal and API Gateway REST APIs of WSO2 API Manager (WSO2 API-M). The `allowedOrigins` property reads values from the system parameters. The defined system parameters are as follows:
+
+| **REST API** | **Parameter** |
+|----------|-----------|
+| Publisher | `rest.api.publisher.allowed.origins` |
+| Developer Portal | `rest.api.devportal.allowed.origins` |
+| API Gateway | `rest.api.gateway.allowed.origins` |
+
+By default, each of the above Product REST APIs allows requests from all origins. However, if you need to only allow a specific set of origins to access a Product REST API, you need to configure the allowed origins as a system parameter for that particular REST API. You can use one of the following methods:
+
+The origin will have the following syntax.
+
+``` tab="Origin Format"
```

Comment:
@tharikaGitHub , are the inverted commas added correctly? It has been added differently in [1]

[1] https://apim.docs.wso2.com/en/3.2.0/develop/product-apis/advanced-configurations/